pym/_emerge/main.py

```diff
@@ -358,6 +359,11 @@ def parse_opts(tmpcmdline, silent=False):
 			"choices" : true_y_or_n
 		},
 
+		"--autounmask-use-only": {
+			"help"    : "only calculate required USE flag changes",
```

Comment:
Can this description be improved a bit? Maybe "write autounmask changes for USE flag changes only" (this would match --autounmask-unrestricted-atoms description)